src/unix/linux_like/linux/mod.rs

```diff
@@ -92,6 +92,16 @@ e! {
     }
 }
 
+c_enum! {
+    e {
```

Comment:
This identifier is the name of the type alias, so should be `pid_type` here. I should add some docs...

Reply:
of course ... thanks :)